webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -102,8 +102,8 @@
     private static BaseFont monoFont, sansSerifFont;
     static {
         try {
-            monoFont = BaseFont.createFont("fonts/LiberationMono-Regular.ttf", BaseFont.IDENTITY_H, BaseFont.EMBEDDED);
-            sansSerifFont = BaseFont.createFont("fonts/wqy-microhei.ttf", BaseFont.IDENTITY_H, BaseFont.EMBEDDED);
+            monoFont = BaseFont.createFont("fonts/segoeui.ttf", BaseFont.IDENTITY_H, BaseFont.EMBEDDED);
+            sansSerifFont = BaseFont.createFont("fonts/segoeui.ttf", BaseFont.IDENTITY_H, BaseFont.EMBEDDED);
```

Comment:
This is concerning to me: is `segoeui` a monospaced font? Does it have support for all the characters we use in all our other translations?

Does `wqy-microhei` not have the Vietnamese characters you need? If we need to find a font that supports every language we do, this might be useful: https://github.com/thewca/tnoodle/pull/246#issuecomment-351934232. Alternatively, we could implement support for using different fonts per language.